docs/man/rpm-common.8.scd

```diff
@@ -0,0 +1,229 @@
+RPM-COMMON(8)
+
+# NAME
+rpm-common - operations and options common to the main *rpm*(8) executables
+
+# SYNOPSIS
+_RPMCMD_ [options] {*-?*|*--help*}
+
+_RPMCMD_ [options] *--showrc*
+
+_RPMCMD_ [options] *--version*
+
+# DESCRIPTION
+The following *rpm* executables share common configuration and the
+options and operations documented in this manual:
+- *rpm*(8)
```

Comment:
I'm thinking if it makes sense to list these. It's easy to get out of sync, and we already link from the relevant commands to this man page.

Reply:
We don't add or remove new commands too often, I doubt it's a whole lot of extra burden. But if we keep the current style in synopsis, this list should actually be used to explain `_RPMCMD_`, and maybe with that it'd make more sense.

But yeah, maybe this is just unnecessary because the relevant man pages already link here.

Reply:
Oh, yup. Either way works for me :smile:

Reply:
I left the command list in place, just referring to the `_RPMCMD_` placeholder. One might land on the rpm-common(8) page directly instead of the other man pages, and it'd seem user-unfriendly to not list which commands this actually corresponds to.